README.md

```diff
@@ -203,41 +203,11 @@ spec:
 
 ## Monitoring
 
-### Install the `kube-prometheus-stack`
-
-Before getting started, you need to ensure that the [`helm`](https://helm.sh/) is installed. Please follow the instructions in the [Installing Helm](https://helm.sh/docs/intro/install/) chapter if you don't have one.
-
-We encourage using the [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack), a helm chart maintained by the community, to install the `Prometheus operator` into the Kubernetes. Follow the instructions below to install:
-
-- Add `prometheus-community` repo
-
-```shell
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm repo update
-```
-
-- Install or upgrade chart
-
-```shell
-helm upgrade --install prometheus prometheus-community/kube-prometheus-stack \
-  -f https://raw.githubusercontent.com/risingwavelabs/risingwave-operator/main/monitoring/kube-prometheus-stack/kube-prometheus-stack.yaml
-```
 
-To check the running status, check the Pods with the following command:
-
-```shell
-kubectl get pods -l release=prometheus
-```
 
-The expected output is like this:
+### Install the `kube-prometheus-stack`
 
-<!-- spellchecker: disable -->
-```plain
-NAME                                                   READY   STATUS    RESTARTS   AGE
-prometheus-kube-prometheus-operator-5f6c8948fb-llvzm   1/1     Running   0          173m
-prometheus-kube-state-metrics-59dd9ffd47-z4777         1/1     Running   0          173m
-prometheus-prometheus-node-exporter-llgp9              1/1     Running   0          169m
-```
+You can install the monitoring stack manually or via the install script. To install it via the script type `./monitoring/install.sh`. If you wish to install it manually, then follow the instructions in the next paragraph
 
 #### Prometheus RemoteWrite (AWS)
```

Comment:
I am not sure what the best approach towards the `RemoteWrite` option is. Should we also install this via the script? Should this be the default? Can we keep it like this? What do you think @arkbriar

Reply:
The remote write option requires extra configurations, such as the AWS access key and secret key. It is a little bit complex to support in the script but still possible. My suggestion is that we can still keep it like this. 

And if you want to try, you can add an option of an environment variable like this and render the value file in the script with `envsubst`:

```shell
ENABLE_REMOTE_WRITE_TO_AWS=1 ./monitoring/install.sh
```

```bash
# ...
envsubst < kube-prometheus-stack/prometheus-remote-write-aws.yaml > /tmp/prometheus-remote-write-aws.yaml
helm upgrade --install -f /tmp/prometheus-remote-write-aws.yaml ...
```

Reply:
[Better still: Use getopts to process params parsed to a script.](https://stackoverflow.com/a/16496491/5345715)